api/client/build.go

```diff
@@ -271,11 +269,7 @@ func validateTag(rawRepo string) (string, error) {
 		return "", err
 	}
 
-	if err := registry.ValidateRepositoryName(ref); err != nil {
-		return "", err
-	}
-
-	return rawRepo, nil
+	return ref.String(), nil
```

Comment:
It seems like the old behavior was to return the `rawRepo` argument. Is there any good reason for this function to return a string instead of just doing validation?